### DISCUSSION TITLE
[Bug] Unable to remove List description

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#9662] [Mobile] [answered]
I tried removing a List description in the iOS app with setting it to empty but it doesn't change to empty and the text stays. Here is a screenrecording:

https://user-images.githubusercontent.com/63184600/148210086-f1131858-a1af-4936-8823-b6d4efd31c32.MOV

I was able to set it to empty on GitHub Web. Tho the app still seems to print an empty description because there is a bigger space after changing it to empty on the Web:

![image](https://user-images.githubusercontent.com/63184600/148210266-1b53c1ba-dc6a-48a8-9a4c-73dff5a0b54c.jpeg)

vs the new List that was created without a description: (if this can be considered an issue)

![image](https://user-images.githubusercontent.com/63184600/148210351-d435d6ff-064e-4b51-b0ae-1d6da78197ea.jpeg)

@brianlovin

**Comment by brianlovin** [comment, marked as answer]
I can reproduce this, thanks so much for the bug report — passing this along to our team now!
